assets/js/mbta.js

```diff
@@ -528,6 +528,48 @@ const stationConfig = {
     },
   ],
   Green: [
+    {
+      id: 'GSCI',
+      name: 'Science Park',
+      zones: {
+        n: false, s: false, e: true, w: true, c: false, m: true,
+      },
+    },
+    {
+      id: 'GNST',
+      name: 'North Station',
+      zones: {
+        n: false, s: false, e: true, w: true, c: false, m: true,
+      },
+    },
+    {
+      id: 'GHAY',
+      name: 'Hay Market',
+      zones: {
+        n: false, s: false, e: true, w: true, c: false, m: false,
+      },
+    },
+    {
+      id: 'GGOV',
+      name: 'Government Center',
+      zones: {
+        n: false, s: false, e: true, w: true, c: false, m: true,
+      },
+    },
+    {
+      id: 'GPRKE',
+      name: 'Park St. East Platform',
```

Comment:
Can we say "Eastbound" and "Westbound" instead of "East" and "West" just to make it really clear that we're referring to the direction of travel?